src/editor/window.rs

```diff
@@ -63,18 +63,22 @@ impl Window {
         });
     }
 
-    pub fn get_cursor_character(&self, window_left: u64, window_top: u64) -> (String, bool) {
+    pub fn get_cursor_character(
+        &self,
+        window_left: u64,
+        window_top: u64,
+    ) -> (String, Option<Arc<Style>>, bool) {
         let character = match self.grid.get_cell(window_left, window_top) {
-            Some((character, _)) => character.clone(),
-            _ => ' '.to_string(),
+            Some((character, style)) => (character.clone(), style.clone()),
+            _ => (' '.to_string(), None),
         };
 
         let double_width = match self.grid.get_cell(window_left + 1, window_top) {
             Some((character, _)) => character.is_empty(),
             _ => false,
         };
 
-        (character, double_width)
+        (character.0, character.1, double_width)
```

Comment:
nit: I think the local variable character here should be renamed. Maybe grid_cell?

Reply:
I agree, I pushed a commit fixing it 👍

Reply:
Gotcha, I renamed:
Cursor struct: "character" -> "grid_cell"
get_cursor_character -> get_cursor_grid_cell